src/ui/screen/screen_mod_manager.cpp

```diff
@@ -203,19 +203,60 @@ bool CScreenModManager::EventProcess(const Event &event)
         pl = static_cast<CList*>(pw->SearchControl(EVENT_INTERFACE_MODLIST_UNLOADED));
         if ( pl == nullptr )  return false;
         modName = pl->GetItemName(pl->GetSelect());
-        modPath = modPathRaw.c_str();
-        boost::filesystem::rename(modPath+OFF+modName, modPath+modName);
-        m_main->ChangePhase(PHASE_MOD_MANAGER);
+
+        if(m_global == false)
+        {
+            modPathRaw = CResourceManager::GetSaveLocation() + "/" + "mods" + "/";
+            modPath = modPathRaw.c_str();
+            boost::filesystem::rename(modPath+OFF+modName, modPath+modName);
+            m_main->ChangePhase(PHASE_MOD_MANAGER);
+        }
+        else
+        {
+            //TODO
+        }
     }
 
     if (event.type == EVENT_INTERFACE_UNLOAD)
     {
         pl = static_cast<CList*>(pw->SearchControl(EVENT_INTERFACE_MODLIST_LOADED));
         if ( pl == nullptr )  return false;
         modName = pl->GetItemName(pl->GetSelect());
-        modPath = modPathRaw.c_str();
-        boost::filesystem::rename(modPath+modName, modPath+OFF+modName);
-        m_main->ChangePhase(PHASE_MOD_MANAGER);
+
+        if(m_global == false)
+        {
+            modPathRaw = CResourceManager::GetSaveLocation() + "/" + "mods" + "/";
+            modPath = modPathRaw.c_str();
+            boost::filesystem::rename(modPath+modName, modPath+OFF+modName);
+            m_main->ChangePhase(PHASE_MOD_MANAGER);
+        }
+        else
+        {
+            //TODO
+        }
+    }
+
+    if (event.type == EVENT_INTERFACE_MODS_DIR)
+    {
+        if(m_global == false)
+        {
+            modPathRaw = CResourceManager::GetSaveLocation() + "/" + "mods";
+            #if defined(PLATFORM_WINDOWS)
+                result = system(("start \""+modPathRaw+"\"").c_str());
+            #elif defined(PLATFORM_LINUX)
+                result = system(("xdg-open \""+modPathRaw+"\"").c_str());
+            #elif defined(PLATFORM_MACOSX)
+                result = system(("open \""+modPathRaw+"\"").c_str());
+            #endif
+            if (result == -1)
+            {
+                GetLogger()->Error("Failed to open Mods Directory! Is directory Exists?\n");
```

Comment:
Shouldn't it be "Does directory exists?"

Reply:
Right...